src/NServiceBus.Gateway/Routing/Sites/ConfigurationBasedSiteRouter.cs

```diff
@@ -4,6 +4,11 @@ namespace NServiceBus.Gateway.Routing.Sites
 
     class ConfigurationBasedSiteRouter : IRouteMessagesToSites
```

Comment:
Just noting that the namespace is off (should be in the root NServiceBus.Gateway namespace). We should clean that up as part of getting the beta out?